config/routes.rb

```diff
@@ -6,4 +6,7 @@
 
   # Uncomment this line to activate the endpoint for the Square hosted checkout workflow.
   # post 'square_checkout', to: '/solidus_square/callback_actions#square_checkout'
+
+  # Uncomment this line to activate the endpoint for the Square order.updated Webhook
```

Comment:
Uncomment it on the config/routes is not enough, the dev should put that line in the routes.rb file of the application.